game-core/src/test/java/games/strategy/ui/ImageScrollerLargeViewTest.java

```diff
@@ -0,0 +1,39 @@
+package games.strategy.ui;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.closeTo;
+import static org.hamcrest.Matchers.is;
+
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+final class ImageScrollerLargeViewTest {
+  @Nested
+  final class NormalizeScaleTest {
+    private static final double ERROR = 1.0E-9;
+    private static final int TILE_SIZE = 256;
+
+    private double normalizeScale(final double value) {
+      return ImageScrollerLargeView.normalizeScale(value, TILE_SIZE);
+    }
+
+    @Test
+    void shouldDiscretizeEachUnitIntoTileSizeValues() {
+      assertThat(normalizeScale(1.0), is(closeTo(1.0, ERROR)));
```

Comment:
IMO the missing context to understand the test expectation is the comment/phrase  the "expected values are multiples of 1/TILE_SIZE"

With that it is pretty clear why we expect oddball values, that those aren't just the actuals necessarily copy/pasted for regression, but actually calculated expectations. I was pondering a custom matcher, it would be a bit nice to help cleanup the ERROR constant, but the readability is pretty good for the most part.